src/python/WMCore/WMSpec/StdSpecs/PromptReco.py

```diff
@@ -61,7 +61,7 @@ def buildWorkload(self):
                                                splitAlgo = self.procJobSplitAlgo,
                                                splitArgs = self.procJobSplitArgs,
                                                stepType = cmsswStepType,
-                                               forceUnmerged = True)
+                                               forceUnmerged = ["write_ALCARECO"] if 'ALCARECO' in self.writeTiers else False)
```

Comment:
and maybe rename the argument from `forceUnmerged` to `forceUnmergedTiers` to make it more clear that it's not a simple bool.